digital-signature/cli/Cargo.toml

```diff
@@ -9,10 +9,10 @@ env_logger = "0.9.0"
 log = "0.4.17"
 digital-signature-methods = { path = "../methods" }
 digital-signature-core = { path = "../core" }
-risc0-zkvm = "0.11"
-risc0-zkp = "0.11"
+risc0-zkvm = { git = "https://github.com/risc0/risc0.git", rev = "65fb3871e3ef468be0a306d9f1adf5631d943b46" }
+risc0-zkp = { git = "https://github.com/risc0/risc0.git", rev = "65fb3871e3ef468be0a306d9f1adf5631d943b46" }
 serde = "1.0"
 sha2 = "0.10.2"
 
 [dev-dependencies]
-ctor = "0.1"
+ctor = "0.1.26"
```

Comment:
So I usually leave off the `z` in `x.y.z` number because my understanding is that cargo doesn't actually use that unless you explicitly pin a specific number with `=x.y.z`

Reply:
good to know, will revise in the in-flight branch this landed in